src/main/java/org/kohsuke/github/GHReason.java

```diff
@@ -0,0 +1,34 @@
+package org.kohsuke.github;
+
+/**
+ * The possible values for reason in verification object from github.
+ * 
+ * @see <a href="https://developer.github.com/v3/repos/commits/#signature-verification-object">List of possible reason
+ *      values</a>
+ * @author Sourabh Sarvotham Parkala
+ */
+public enum GHReason {
+    EXPIRED_KEY("expired_key"),
+    NOT_SIGNING_KEY("not_signing_key"),
+    GPGVERIFY_ERROR("gpgverify_error"),
+    GPGVERIFY_UNAVAILABLE("gpgverify_unavailable"),
+    UNSIGNED("unsigned"),
+    UNKNOW_SIGNATURE_TYPE("unknown_signature_type"),
+    NO_USER("no_user"),
+    UNVERIFIED_EMAIL("unverified_email"),
+    BAD_EMAIL("bad_email"),
+    UNKNOWN_KEY("unknown_key"),
+    MALFORMED_SIGNATURE("malformed_signature"),
+    INVALID("invalid"),
+    VALID("valid");
+
+    private final String value;
```

Comment:
Not sure if `value` is necessary.

Reply:
@artem-smotrakov 
In general, you'd be correct.  But I think the default mapping for enums is all-caps and convert-underscores-to-dashes. Which would result in `EXPIRED_KEY` converting to `expired-key` and vice versa.  Once tests are added, we can see whether this is needed or not.